mutation.go

```diff
@@ -125,9 +130,10 @@ const (
 
 // Mutate stores mutation instruction.
 type Mutate struct {
-	Type  ChangeOp
-	Field string
-	Value interface{}
+	Type     ChangeOp
+	Field    string
+	Value    interface{}
+	NoReload Reload
```

Comment:
Skip reload for queries that use SQL expressions which can be evaluated without querying